packages/protocol/contracts/L1/libs/LibVerifying.sol

```diff
@@ -39,10 +39,6 @@ library LibVerifying {
         uint8 contestations
     );
 
-    event CrossChainSynced(
```

Comment:
This event is no longer necessary as the following event in SignalService is good enough:

```solidity
 event ChainDataRelayed(
        uint64 indexed chainid,
        uint64 indexed blockId,
        bytes32 indexed kind,
        bytes32 data,
        bytes32 signal
    );```